src/Workspaces/Remote/Core/Services/SolutionCreator.cs

```diff
@@ -355,6 +359,8 @@ private async Task<Project> UpdateProjectInfoAsync(Project project, Checksum inf
                 Contract.ThrowIfTrue(oldDocumentChecksums.Checksum == newDocumentChecksums.Checksum);
 
                 var document = project.GetDocument(documentId) ?? project.GetAdditionalDocument(documentId) ?? project.GetAnalyzerConfigDocument(documentId);
```

Comment:
Not your change but https://github.com/dotnet/roslyn/blob/87c9fff2427ddc02d36ce165789516f2bea85a2a/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/ISolutionExtensions.cs#L30 exists, if you care.